package.json

```diff
@@ -6,8 +6,10 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "lodash": "^4.17.19",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-native": "^0.63.2",
```

Comment:
Why did you add react-native?

Reply:
Nice catch, hmm I think it got added when I did an npm install.
I was having some problems with using the useState hooks inside of functional components and decided to delete the whole nodule_modules folder. This might have caused that.
(Turns out the problem with using the hooks was because the functional components have to start with a capital letter, took me a while to figure that out